modules/data_warehouse/bigquery.tf

```diff
@@ -0,0 +1,334 @@
+  ]
+}
+
+# Load Queries for Stored Procedure Execution
+# # Load Lookup Data Tables
+resource "google_bigquery_routine" "sp_provision_lookup_tables" {
+  project         = module.project-services.project_id
+  dataset_id      = google_bigquery_dataset.ds_edw.dataset_id
+  routine_id      = "sp_provision_lookup_tables"
+  routine_type    = "PROCEDURE"
+  language        = "SQL"
+  definition_body = templatefile("${path.module}/src/sql/sp_provision_lookup_tables.sql", { project_id = module.project-services.project_id })
+
+  depends_on = [
+    google_bigquery_dataset.ds_edw,
+  ]
+}
+
+
+# # Add Looker Studio Data Report Procedure
+resource "google_bigquery_routine" "sproc_sp_demo_datastudio_report" {
+  project         = module.project-services.project_id
+  dataset_id      = google_bigquery_dataset.ds_edw.dataset_id
+  routine_id      = "sp_lookerstudio_report"
+  routine_type    = "PROCEDURE"
+  language        = "SQL"
+  definition_body = templatefile("${path.module}/src/sql/sp_lookerstudio_report.sql", { project_id = module.project-services.project_id })
+
+  depends_on = [
+    google_bigquery_table.tbl_edw_taxi,
+  ]
+}
+
+# # Add Sample Queries
+resource "google_bigquery_routine" "sp_sample_queries" {
+  project         = module.project-services.project_id
+  dataset_id      = google_bigquery_dataset.ds_edw.dataset_id
+  routine_id      = "sp_sample_queries"
+  routine_type    = "PROCEDURE"
+  language        = "SQL"
+  definition_body = templatefile("${path.module}/src/sql/sp_sample_queries.sql", { project_id = module.project-services.project_id })
+
+  depends_on = [
+    google_bigquery_table.tbl_edw_taxi,
+  ]
+}
+
+# # Add Bigquery ML Model
+resource "google_bigquery_routine" "sp_bigqueryml_model" {
+  project         = module.project-services.project_id
+  dataset_id      = google_bigquery_dataset.ds_edw.dataset_id
+  routine_id      = "sp_bigqueryml_model"
+  routine_type    = "PROCEDURE"
+  language        = "SQL"
+  definition_body = templatefile("${path.module}/src/sql/sp_bigqueryml_model.sql", { project_id = module.project-services.project_id })
+
+  depends_on = [
+    google_bigquery_table.tbl_edw_taxi,
+  ]
+}
+
+# # Add Translation Scripts
+resource "google_bigquery_routine" "sp_sample_translation_queries" {
+  project         = module.project-services.project_id
+  dataset_id      = google_bigquery_dataset.ds_edw.dataset_id
+  routine_id      = "sp_sample_translation_queries"
+  routine_type    = "PROCEDURE"
+  language        = "SQL"
+  definition_body = templatefile("${path.module}/src/sql/sp_sample_translation_queries.sql", { project_id = module.project-services.project_id })
+
+  depends_on = [
+    google_bigquery_table.tbl_edw_taxi,
+  ]
+}
+
+# Add Scheduled Query
+# # Set up DTS permissions
+resource "google_project_service_identity" "bigquery_data_transfer_sa" {
+  provider = google-beta
+  project  = module.project-services.project_id
+  service  = "bigquerydatatransfer.googleapis.com"
+}
```

Comment:
You can directly activate and get this via project services

https://github.com/terraform-google-modules/terraform-google-project-factory/blob/de1c1a1b4985c67002b2f2d37ba8a3bb7221167d/modules/project_services/variables.tf#L34

https://github.com/terraform-google-modules/terraform-google-project-factory/blob/de1c1a1b4985c67002b2f2d37ba8a3bb7221167d/modules/project_services/outputs.tf#L27

Reply:
Added a bug for this